Documentation/ReleaseNotes.md

```diff
@@ -364,7 +364,7 @@ For the smoothest upgrade path, please use the following steps.
 1. Close Unity and Delete the **Library** folder. This step is necessary to force Unity to refresh its
    asset database and reconcile existing custom profiles.
 1. Launch Unity, and for each scene in the project
-    - Delete **MixedRealityToolkit** and **MixedRealityPlayspace**, if present, from the hierarchy
+    - Delete **MixedRealityToolkit** and **MixedRealityPlayspace**, if present, from the hierarchy. This will delete the main camera, but it will be re-created in the next step. If you've changed any properties of the main camera, you may want to note those changes and re-apply them.
```

Comment:
```suggestion
    - Delete **MixedRealityToolkit** and **MixedRealityPlayspace**, if present, from the hierarchy. This will delete the main camera, but it will be re-created in the next step. If any properties of the main camera have been manually changed, these will have to be re-applied manually once the new camera is created.
```

Reply:
Avoid use of "you". I know a lot of MRTK documentation does not entirely follow this, but trying to move away from the informal

Reply:
TBH, while I agree with being formal, I disagree that "you" is informal.  I think it is important to avoid passive voice, too.  However I'll do as you've asked.